<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>900</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CRASH LINUX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>960</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>AUTO DRIVE</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>811</width>
      <height>741</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="speed_guage"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1131</x>
      <y>110</y>
      <width>741</width>
      <height>741</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="rpm_guage"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>838</x>
      <y>680</y>
      <width>261</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="car"/>
   </widget>
   <widget class="QTextEdit" name="am">
    <property name="geometry">
     <rect>
      <x>851</x>
      <y>570</y>
      <width>221</width>
      <height>75</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#ffffff;&quot;&gt;AUTO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>721</x>
      <y>960</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="fuel"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>781</x>
      <y>960</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="engine"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>661</x>
      <y>960</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="abs"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>1104</x>
      <y>959</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="airbag"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>1161</x>
      <y>959</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="seatbelt"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_10">
    <property name="geometry">
     <rect>
      <x>1221</x>
      <y>959</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="oil"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_11">
    <property name="geometry">
     <rect>
      <x>841</x>
      <y>178</y>
      <width>281</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="radar_graph"/>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>921</x>
      <y>515</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
